x/incentives/spec/02_state.md

```diff
@@ -0,0 +1,70 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Incentives management
+
+All the incentives that is going to be provided are locked into `IncentivePool` until it's released to the users after a specific period of time.
+
+### Pot
+
+Rewards to be distributed are organized by `Pot`. The `Pot` describes how users can get reward, stores the amount of coins in the pot, the cadence at which rewards are to be distributed, and the number of epochs to distribute the reward over.
+
+```go
+type LockType int
+const (
+  ByLockDuration LockType = iota // locks which has more than specific duration
+  ByLockTime // locks which are started before specific time
+)
+
+type DistrCondition struct {
+  LockType LockType // type of lock condition
+  Denom  string // lock denom
+  Duration time.Duration // condition for lock duration, only valid if positive
+  Timestamp time.Time // condition for lock start time, not valid if unset value
+}
+
+type Pot struct {
+  ID           uint64 // unique ID of a Pot
+  DistributeTo []DistrCondition // distribute condition of a lock which meet one of these conditions
```

Comment:
We need some mechanism to encode the percent amount to distribute funds to. Perhaps a "weight" field should be added to each DistrCondition, and a "total weight" field added to pot?

Reply:
Oh interesting.  I was thinking that we would create separate pots for each distribution condition.  Is there any benefit to having multiple distribution conditions in a single pot?

Reply:
yeah, adding different pot per distr condition would be simpler.

Reply:
Ok, so this shouldn't be a slice, it should just be a single DistrCondition right?